packages/neos-ui/src/Containers/Modals/SelectNodeType/index.js

```diff
@@ -165,6 +165,17 @@ export default class SelectNodeType extends PureComponent {
 
     handleNodeTypeFilterChange = filterSearchTerm => this.setState({filterSearchTerm});
 
+    componentDidUpdate() {
+        const {insertMode} = this.state;
+        if (insertMode === 'into' &&
+            this.getAllowedNodeTypesByCurrentInsertMode().length === 1 &&
+            this.getAllowedNodeTypesByCurrentInsertMode()[0].nodeTypes.length === 1) {
+            setTimeout(() => {
```

Comment:
If timeout actually needed here?

Reply:
Without overruling the Modal did not work.
Hacky I know.. but in the worst case (super slow Computer) nothing should happen and the normal Modal is opening.

Reply:
That's super strange actually... Would be cool to figure out why it happened, but I guess it's ok to leave it like this if you got no time.

Reply:
Could you at least leave a comment here to justify the usage of setTimeout?

Reply:
@dimaip okay I checked it again. It was a problem with the asynchronous setState.
This is solved now in the latest commit by executing the check in the render function itself.